configs/atlassian-backup.conf

```diff
@@ -27,5 +42,17 @@ CONFLUENCE_BACKUP_DIR='/atlassian/backups/confluence'
 CONFLUENCE_ATTACHMENTS_PATH='/local/jira/confluence-5.7.1/attachments'
 CONFLUENCE_CONFIGURATION_PATH='/local/jira/confluence-5.7.1/confluence.cfg.xml'
 
+# Confluence DB Connection Information
+# The following vars are used to connect to your Confluence DB instance. 
+# The mysql information is only used for MySQL connections, however CONFLUENCE_DB_NAME is used for both PG and MySQL.
+# Chose the backend for your database, should be either 'pg' for PostgreSQL or 'mysql' for MySQL.
+CONFLUENCE_DB_BACKEND='pg'
+# CONFLUENCE_DB_BACKEND='mysql'
+# A database user that has access to your Confluence MySQL DB
+CONFLUENCE_MYSQL_USER=''
+# The password for the user that has access to your Confluence MySQL DB
+CONFLUENCE_MYSQL_PASS=''
+# The MySQL host that hosts your Confluence DB
+CONFLUENCE_MYSQL_HOST=''
 # Confluence Database
 CONFLUENCE_DB_NAME='confluencedb_5_7_1'
```

Comment:
I recommend (but I do not demand) removing "_5_7_1". Database names in Atlassian documentation do not include the version, but I appended it anyway.

As it turns out, it is unnatural to change the database name when upgrading, so Atlassian is right.

Reply:
I've removed the version numbers from the db names. I also updated the Jira and Confluence home paths to more recent defaults.